app/components/auth/signUp/components/finalSignUpContent/index.tsx

```diff
@@ -18,29 +19,49 @@ function withSocialCompleteContent() {
   );
 }
 
-function withEmailCompleteContent() {
+function withEmailCompleteContent(email: string) {
   return (
     <>
       <div className={s.finalSignUpContent}>{`Please check your email
 and verify your email address`}</div>
       <Icon className={s.finalSignUpIconWrapper} icon="VERIFICATION_EMAIL_ICON" />
+      <div className={s.toEmail}>
+        Sent to <span className={s.email}>{email}</span>
+      </div>
     </>
   );
 }
 
 const FinalSignUpContent: React.FunctionComponent<FinalSignUpContentProps> = props => {
-  const { onSubmit, contentType } = props;
+  const { onSubmit, contentType, email } = props;
 
-  const finalContent = contentType === 'email' ? withEmailCompleteContent() : withSocialCompleteContent();
+  const finalContent = contentType === 'email' ? withEmailCompleteContent(email!) : withSocialCompleteContent();
```

Comment:
I strongly recommend the below coding style. 
please avoid using bang(!) type converter as possible as you can.

```
let content = withSocialCompleteContent();
if (contentType === 'email' && !!email) {
  withEmailCompleteContent(email);
}
```